src/argilla/client/feedback/schemas.py

```diff
@@ -423,8 +432,38 @@ class MultiLabelQuestion(_LabelQuestion):
     settings: Dict[str, Any] = Field({"type": "multi_label_selection"})
 
 
+class RankingQuestion(QuestionSchema):
+    """Schema for the `RankingQuestion` question-type.
+
+    Args:
+        settings: The settings for the question, including the type and options.
+        values: The values for the question, to be formatted and included as part of
+            the settings.
```

Comment:
can we add here an example like in the rest of questions?

Reply:
Sure thing!